src/integ-test/bdd-test/features/primitives/paGetPaymentRes_syntax_ko.feature

```diff
@@ -0,0 +1,288 @@
+      | retentionDate               | 12-28-2022                                                                                                                                                                                                                                                        | SIN_PGPR_28   |
+      | retentionDate               | 12-09-22                                                                                                                                                                                                                                                          | SIN_PGPR_28   |
+      | retentionDate               | 12-08-2022T12:00:678                                                                                                                                                                                                                                              | SIN_PGPR_28   |
+      | lastPayment                 | Empty                                                                                                                                                                                                                                                             | SIN_PGPR_30   |
+      | lastPayment                 | 2                                                                                                                                                                                                                                                                 | SIN_PGPR_31   |
+      | description                 | None                                                                                                                                                                                                                                                              | SIN_PGPR_32   |
+      | description                 | Empty                                                                                                                                                                                                                                                             | SIN_PGPR_33   |
+      | description                 | test di prova per una lunghezza superiore a 140 caratteri alfanumerici, per verificare che il nodo risponda PPT_STAZIONE_INT_PA_ERRORE_RESPONSE                                                                                                                   | SIN_PGPR_34   |
+      | paymentCategory             | None                                                                                                                                                                                                                                                              | SIN_PGPR_35   |
```

Comment:
`paymentCategory` is not present in the `paGetPayment` XML

Reply:
You're right, in the latest version of the paGetPayment XML this tag has been deleted